README.md

```diff
@@ -51,4 +51,5 @@ Click outside of textboxes before placing or removing barrels.
 
 **Aryamanarora** - Metadata
 
-**Ericw31415** - Tidying up & improving info stuff
+# (Note: ericw31415 doesn't like having his username capitalized.)
+**ericw31415** - Tidying up & improving info stuff
```

Comment:
How do you feel about the separation? Err, between large contributors and minor ones. I've been uncomfortable with the that distinction since I added it, but you appear to be doing cleanup right now and I wanted to get your opinion on it.

I feel there needs to be some distinction made, such as comparing blob(creator) to the rest of us, but I don't have a clear definition to work on. The current definition basically is "If they come back, move them up" but that's really poor and prone to a ton of bias and can create issues.